src/ui.rs

```diff
@@ -189,34 +189,18 @@ impl From<crossterm::event::Event> for Event {
                 modifiers: _,
             }) => Self::ScrollDown,
 
-            Event::Key(
-                KeyEvent {
-                    code: KeyCode::PageUp,
-                    modifiers: KeyModifiers::NONE,
-                    kind: KeyEventKind::Press,
-                    state: _,
-                }
-                | KeyEvent {
-                    code: KeyCode::Char('b'),
-                    modifiers: KeyModifiers::CONTROL,
-                    kind: KeyEventKind::Press,
-                    state: _,
-                },
-            ) => Self::PageUp,
-            Event::Key(
-                KeyEvent {
-                    code: KeyCode::PageDown,
-                    modifiers: KeyModifiers::NONE,
-                    kind: KeyEventKind::Press,
-                    state: _,
-                }
-                | KeyEvent {
-                    code: KeyCode::Char('f'),
-                    modifiers: KeyModifiers::CONTROL,
-                    kind: KeyEventKind::Press,
-                    state: _,
-                },
-            ) => Self::PageDown,
+            Event::Key(KeyEvent {
+                code: KeyCode::PageUp,
+                modifiers: KeyModifiers::CONTROL,
+                kind: KeyEventKind::Press,
+                state: _,
+            }) => Self::PageUp,
+            Event::Key(KeyEvent {
+                code: KeyCode::PageDown,
+                modifiers: KeyModifiers::CONTROL,
```

Comment:
question: Do you happen to know if ctrl-fn-up works to trigger this on Mac? (I don't have any reason to assume it won't work, but I haven't checked.)

Reply:
Yeah it seems to work based on my testing. My guess is that the fn key isn't exposed by the os as a keystroke to the application.